qa/L0_perf_analyzer_report/test.sh

```diff
@@ -166,7 +160,7 @@ set -e
 
 # Cleanup
 kill $SERVER_PID
-wait $SERVER_PID
```

Comment:
Why is this removed?

Reply:
I see you mentioned this:
> Removed wait $SERVER_PID as is not returning/exiting with 0 because Triton server is taking a long time to shut down.

Can you elaborate on "long time" here? If it's slow, maybe it's expected. If it's hanging, there may be a bug.